Dockerfile

```diff
@@ -11,14 +11,12 @@
 
 WORKDIR /app
 
-ENV PYTHONUNBUFFERED=1
+ENV PYTHONUNBUFFERED="default"
+ENV ENV_ENGINE="default"
+ENV ENV_NAME="default"
+ENV ENV_USER="default"
+ENV ENV_PASSWORD="default"
```

Comment:
The `ENV_PASSWORD` variable is set to 'default'. This poses a security risk as it may expose sensitive information. Consider using Docker secrets or setting this variable at runtime.